Mathlib/Topology/Algebra/Algebra.lean

```diff
@@ -74,16 +78,471 @@ theorem algebraMapCLM_toLinearMap : (algebraMapCLM R A).toLinearMap = Algebra.li
   rfl
 #align algebra_map_clm_to_linear_map algebraMapCLM_toLinearMap
 
+/-- If `R` is a discrete topological ring, then any topological ring `S` which is an `R`-algebra
+  is also a topological `R`-algebra. -/
+instance DiscreteTopology.continuousSMul [TopologicalSemiring A] [DiscreteTopology R] :
+    ContinuousSMul R A := continuousSMul_of_algebraMap _ _ continuous_of_discreteTopology
+
 end TopologicalAlgebra
 
-section TopologicalAlgebra
+section TopologicalAlgebra --
+section
 
 variable {R : Type*} [CommSemiring R]
 variable {A : Type u} [TopologicalSpace A]
 variable [Semiring A] [Algebra R A]
 
 #align subalgebra.has_continuous_smul SMulMemClass.continuousSMul
 
+end
+section
```

Comment:
I don't understand what is happening here

Reply:
We did not add this, but looking at the [git blame](https://github.com/leanprover-community/mathlib4/commit/350879f78bdda0673c10fb75895e4492b226b3d2), it seems that `Submodule.continuousSMul` was superseded by
`SMulMemClass.continuousSMul` and this line is recording the new align.